frontend/pages/checkOut.vue

```diff
@@ -173,32 +185,42 @@ export default {
         consent: 'no'
       },
       count: 0,
-      price: 1,
+      price: this.$store.getters['getPrice'],
       isCheckoutVisible: false,
       laoding: false
     };
   },
+  setup() {
+    const isOpen = ref(false);
+
+    const toggleDropdown = () => {
+      isOpen.value = !isOpen.value;
+    };
+
+    return { isOpen, toggleDropdown };
+  },
   methods: {
     submitForm(e) {
       e.preventDefault();
       this.loading = true;
       let email = this.$store.getters['auth/userEmail'];
       console.log("email", email)
+      console.log(this.formValue)
       const data = {
         "first_name": this.formValue.firstName,
         "last_name": this.formValue.lastName,
-        "email": "priyanshmehta61@gmail.com",
+        "email": this.formValue.email,
         "hostel_address": this.formValue.hostel_address,
         "roll": this.formValue.roll,
         "branch": this.formValue.branch,
         "club_member": this.formValue.club_member,
         "phone": this.formValue.phone,
         "consent": this.formValue.consent,
-        "comment": this.formValue.comment,
-        "price": this.price,
+        "comment": "fdsdf",
+        "price": this.$store.getters['getPrice'],
         "prod_name": "needs to be added dynamically",
         "prod_type": "ticket",
-        "prod_id": "1", // either the event id or the product id not the ticket id
+        "prod_id": "2", // either the event id or the product id not the ticket id
```

Comment:
No `prod_id` given here, pass this here from the frontend

Reply:
I don't know as initially only one product will be there and how can I pass it from frontend, isn't prod_id should be unique and linked to product in backend??